src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ToGerritRunListener.java

```diff
@@ -428,15 +429,19 @@ private String obtainFailureMessage(@Nullable GerritTriggeredEvent event,
                 // The filename may contain environment variables
                 filepath = envVars.expand(filepath);
 
-                // Check for ANT-style file path
-                FilePath[] matches = this.getMatchingWorkspaceFiles(build.getWorkspace(), filepath);
-                logger.debug("Found matching workspace files: {}", matches);
+                if (build instanceof AbstractBuild) {
```

Comment:
I's not possible and I'm not sure what would be involved doing that for workflow. I think we can leave that for now and come back to it as a separate task.